### DISCUSSION TITLE
Removing a site should not delete all the local files!

### DISCUSSION THREAD

**Discussion by photonstorm** [beyondcode/herd-community#1136] [Ideas]
When using Herd Pro (latest version), if I want to edit a site it's currently impossible. You can't edit the URL or the path it is mapped to, or anything about it. What is worse is that you can't even delete the entry from Herd without it removing the whole folder of contents as well! This seems like a massive oversight. At the very least, please allow us to delete a Herd entry without nuking our files, so we can re-create it with the updated properties that we can't currently set directly due to the lack of an edit feature.

**Comment by mpociot** [comment]
What exactly do you want to edit?

The "parked sites" feature automatically maps all folder names to URLs. If you want to change how a folder gets linked to an URL, you may use the `herd link` command.

This allows you to specify a custom domain for your site. For example:

```bash
cd my-site
herd link my-custom-url
```

Now the project `my-site` is available as `my-custom-url.test`

**Comment by photonstorm** [comment]
Yes, I do this as well because the parked sites feature is not very useful, to be honest (it's super-rare to want to serve the folder of a project, much more common to need to map to a sub-folder like `public`).

It's just a shame you can't edit a site/promote a parked site to a 'full site', or unlink a custom linked site via the UI. To do any edits you need to reach for the CLI - which is basically just using Valet at that point.

**Comment by Noaber** [comment]
Agree with @photonstorm : Today I deleted a site in the GUI and all the files are also deleted.
My assumption was that I "unlink" the site and could recreate it.

Maybe it's possible to get a choice to delete the site without local files ? or just an unlink button, otherwise I still have to do this in the CLI (herd unlink) : @mpociot

**Reply by Seandalf** [reply]
Hard agree - it's not intuitively laid out in the UI that deleting a site from the site list will also remove all local files. Especially if you added the site as an existing project, why would I then want Herd to delete local files at all?

There should at least be the choice between full delete (including local files) and only deleting the link/the project from Herd.
